### ISSUE TITLE
Guangdong (Early Game) - Max PTRG Profits not appearing

### ISSUE THREAD

**Issue by Theeserras** [TNODevs/TNO-Bug-Tracker#3269]
### Operating System

Windows 11

### HOI4 version

1.14.5

### TNO version

1.5.0

### Do you have other mods in use?

The Moe Order, Last Waltz

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

MTG, BBA, AAT

### Files

N/A

### Describe the bug

The bug happened for some time now, everytime I booted up a game to play Guangdong, I get to the PTRG part of the game, and yet the max spirit, where I could get the most amount of profits, does not seem to work.did

### Steps to reproduce / Cause of the issue

There are no special steps. As stated, I played normally.

### Expected behaviour

Basically, I expected for what the event stated should happen, and that is I get the national spirit, "PTRG profits".

### Screenshots

https://github.com/TNODevs/TNO-Bug-Tracker/assets/171581783/e1079003-e875-4ddd-9135-df526fb2e8ea

**Comment by Jack121Q** [comment]
It says on the button you get it in 30 days.
